content/posts/hiero_links_of_august_twentyfour.md

```
@@ -0,0 +1,29 @@
+Below are links to some of the things that happened this past week. **Check it out!**
+
+---
+**[Achintya S](https://github.com/achintya2k5)** added the[GetFileContents RPC](https://github.com/hiero-ledger/hiero-sdk-python/pull/2539) method to the TCK module in the Python SDK. 
+
+**[Daniel N](https://github.com/danielmarv)** added a [Navbar](https://github.com/hiero-ledger/hiero-website/pull/569) update to the Hiero Website to start the website redesign efforts.
+
+**[Angelina C](https://github.com/aceppaluni)** added [Network Address Book support](https://github.com/hiero-ledger/hiero-enterprise-java/pull/251) to the Enterprise-Java repository.
+
+**[Andrew B](https://github.com/andrewb1269)** updated the default run time and input descriptions for [MDLT Adhoc runs](https://github.com/hiero-ledger/hiero-consensus-node/pull/27000) in the Hiero Consensus Node. 
+
+**Want to be featured on the Hiero-Website?**
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)